contracts/src/v0.8/automation/dev/test/AutomationRegistry2_3.t.sol

```diff
@@ -875,6 +883,9 @@ contract NOPsSettlement is SetUp {
       assertEq(i, index);
       assertEq(0, balance);
     }
+
+    // assert reserve amount of LINK is reduced by the offchain settlement amount
+    assertEq(registry.getReserveAmount(address(linkToken)), prevReserveBalances - totalPayments);
```

Comment:
reserve amount of link should be 0 since it is offchain mode, users cant deposit link to registry

add test to cover the above